gradle/docker.gradle

```diff
@@ -3,7 +3,7 @@ import groovy.time.*
 /**
  * Utility to build docker images based in gradle projects
  *
- * This extends gradle's 'application' plugin logic with a 'buildBinaries' task which builds
```

Comment:
Can this file be deleted since the Dockerfile was removed?

Reply:
Not yet, because it's still used to build the images for 'incubator-openwhisk' in support of the testing.  In time we can try to get the build to use the docker.gradle in 'incubator-openwhisk', but I'm trying to manage the number of moving parts.

Reply:
From test_openwhisk.sh:

```
+cd $OPENWHISK_HOME/ansible
 +ANSIBLE_CMD="ansible-playbook -i environments/local -e docker_image_prefix=testing"
```

Doesn't the above mean the docker.gradle file from the OpenWhisk directory is being used?